lib/sync-metrics.js

```diff
@@ -72,7 +72,7 @@ var aggregateData = function(metric, records) {
       return record.tags[metric.groupByTag];
     }).reduce(function(reduced, groupRecords, groupKey) {
       var processedData = _.reduce(groupRecords, function(memo, groupRecord) {
-        groupRecords = _.sortBy(groupRecords, 'ts');
+        groupRecords = _.sortBy(groupRecords, 'ts').reverse();
```

Comment:
I think this line should be moved up, before L74. It should be like this:

```
groupRecords = _.sortBy(groupRecords, 'ts').reverse();
var processedData = _.reduce(groupRecords, function(memo, groupRecord) {})
...

```

The idea is to have the records sorted before running the reduce. We should only need to sort it once. Not sure how we end up this code.

Also removing `.reverse()` is right. For some reason I thought the results are in descending order.